src/util/symbol_table.h

```diff
@@ -109,6 +109,10 @@ class symbol_tablet : public symbol_table_baset
   {
     return iteratort(internal_symbols.end());
   }
+
+  void validate() const
+  {
```

Comment:
UNIMPLEMENTED?

Reply:
or remove

Reply:
I'm working on a PR that will add a body, so I'd leave it empty.